gradle.properties

```
@@ -1,7 +1,7 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-enigma_version=0.27.3
+enigma_version=1.3.2
```

```suggestion
enigma_version=1.0.0
```
Version 1.0.0 is the last one with supports Java below 16

---

Doesn't matter for Yarn. None of its dependencies are (should be) inherited. It's purely for mappings. Using The latest version means I get to use all the newest features.

---

Oh, *sigh*. Right. That means that the publish workflow will have to use JDK16, which will generate incompatible bytecode...

---

As long as we're not using Yarn for patches, that shouldn't be a problem though...

---

Hmmm, it looks like something else is causing a problem. I tried reverting Enigma, Gradle, and JDK locally, but build is still failing at [build.gradle line 483](https://github.com/Better-Than-Updates-MC/Mappings/blob/pr/1.6.4/update/build.gradle#L483)

---

for me too... are the intermediary broken??

---

Unlikely. Everything was working up to this last couple of commits.